e2e/tests/localAuthorityMaintainsCaseAfterSubmission_test.js

```diff
@@ -50,3 +51,41 @@ Scenario('local authority uploads court bundle', (I, uploadDocumentsEventPage, s
   caseViewPage.selectTab(caseViewPage.tabs.documents);
   I.seeDocument('Court bundle', 'mockFile.txt');
 });
+
+Scenario('local authority provides a statements of service', async (I, caseViewPage, loginPage, provideStatementOfServiceEventPage) => {
+  caseViewPage.goToNewActions(config.administrationActions.provideStatementOfService);
+  await provideStatementOfServiceEventPage.enterRecipientDetails(recipients[0]);
+  provideStatementOfServiceEventPage.addRecipient();
+  await provideStatementOfServiceEventPage.enterRecipientDetails(recipients[1]);
+  provideStatementOfServiceEventPage.giveDeclaration();
+  I.continueAndSave();
+  I.seeEventSubmissionConfirmation(config.administrationActions.provideStatementOfService);
+  caseViewPage.selectTab(caseViewPage.tabs.legalBasis);
+  I.seeAnswerInTab(1, 'Recipients 1', 'Name of recipient', recipients[0].name);
+  I.seeAnswerInTab(2, 'Recipients 1', 'Do you have the recipient\'s address?', recipients[0].addressCheck);
+  I.seeAnswerInTab(1, 'Recipient\'s address', 'Building and Street', recipients[0].address.buildingAndStreet.lineOne);
+  I.seeAnswerInTab(2, 'Recipient\'s address', 'Address Line 2', recipients[0].address.buildingAndStreet.lineTwo);
+  I.seeAnswerInTab(3, 'Recipient\'s address', 'Address Line 3', recipients[0].address.buildingAndStreet.lineThree);
+  I.seeAnswerInTab(4, 'Recipient\'s address', 'Town or City', recipients[0].address.town);
+  I.seeAnswerInTab(5, 'Recipient\'s address', 'Postcode/Zipcode', recipients[0].address.postcode);
+  I.seeAnswerInTab(6, 'Recipient\'s address', 'Country', recipients[0].address.country);
+  I.seeAnswerInTab(4, 'Recipients 1', 'Documents', recipients[0].documents);
```

Comment:
What happened to 3?

Reply:
So it sees the recipients address as the third element and then when it goes back to recipients 1 it is the next in the list so it is 4